css/loginStyles.css

```diff
@@ -0,0 +1,69 @@
+body {
```

Comment:
Best to define the box-sizing at the beginning (if you are using a box sizing ting)